src/custom-surf/components/cim/OpenForm.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019-2022 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import UserInputFormWizard from "components/inputForms/UserInputFormWizard";
+import { ServiceTicketProcessState } from "custom/types";
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import ApplicationContext from "utils/ApplicationContext";
+import { setFlash } from "utils/Flash";
+import { Form, FormNotCompleteResponse } from "utils/types";
+
+interface IProps {
+    formKey: string;
+    ticket_id: string;
+    handleSubmit: (userInputs: any) => void;
+    handleCancel: () => void;
+}
+
+export default function OpenForm(props: IProps) {
+    const intl = useIntl();
+    const { formKey, ticket_id, handleSubmit, handleCancel } = props;
```

Comment:
can be done directly within the function
`export default function OpenForm({ formKey, ticket_id, handleSubmit, handleCancel }: IProps) {`